cpp/include/cudf/copying.hpp

```diff
@@ -367,6 +367,7 @@ std::vector<table_view> slice(table_view const& input,
  * `splits[i] != input.size()`, or `[splits[i-1], splits[i]]` otherwise.
  *
  * For all `i` it is expected `splits[i] <= splits[i+1] <= input.size()`
+ * For a `splits` size N, there will always be N+1 splits in the output
```

Comment:
Can you update the above documentation that says `splits[i] != input.size()`

Reply:
Ahh, missed that.